docs/reference/elements/py-config.md

```diff
@@ -212,10 +217,17 @@ The following optional values are supported by `<py-config>`:
 | `license` | string | License to be used for the user application. |
 | `autoclose_loader` | boolean | If false, PyScript will not close the loading splash screen when the startup operations finish. |
 | `packages` | List of Packages | Dependencies on 3rd party OSS packages are specified here. The default value is an empty list. |
-| `paths` | List of Paths | Local Python modules are to be specified here. The default value is an empty list. |
+| `fetch` | List of Stuff to fetch | Local Python modules OR resources from the internet are to be specified here using a Fetch Configuration, described below. The default value is an empty list. |
 | `plugins` | List of Plugins | List of Plugins are to be specified here. The default value is an empty list. |
 | `runtimes` | List of Runtimes | List of runtime configurations, described below. The default value contains a single Pyodide based runtime. |
 
+A fetch configuration consists of the following:
+| Value | Type | Description |
+| ----- | ---- | ----------- |
+| `from` | string | Base URL for the resource to be fetched. |
+| `to_folder` | string | Name of the folder to create in the filesystem. |
+| `files` | List of string | List of files to be downloaded. |
+
```

Comment:
I think it's fine for this PR, but we should have better docs to explain how this works.
@tedpatrick do we have a process to signal "please write better docs for this topic" so that our technical writers know about it?
E.g. by putting `TODO: bla bla bla` comments directly inside the docs or creating an issue?

Reply:
I guess you cannot see an entry for `to_file` here as well since I think you reviewed a previous commit.